### PR TITLE
TESB-31889 Jackson security update to 2.11.4.

### PR DESCRIPTION
**Pull request by andreasmattes**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -40,12 +40,13 @@
...
 
   <properties>
-    <apache-camel.features.tesb.version>2.24.2.tesb7</apache-camel.features.tesb.version>
+    <apache-camel.features.tesb.version>2.24.2.tesb8</apache-camel.features.tesb.version>
     <camel-mqtt.tesb.version>2.24.2.tesb1</camel-mqtt.tesb.version>
     <cxf.tesb.version>3.3.4</cxf.tesb.version>
     <woodstox-core.tesb.version>5.0.3</woodstox-core.tesb.version>
     <abdera-parser.tesb.version>1.1.3_2</abdera-parser.tesb.version>
-    <jackson2.tesb.version>2.10.1</jackson2.tesb.version>
+    <jackson1.tesb.version>1.9.15-TALEND</jackson1.tesb.version>
```
Comment (jgreffe) [inline pom.xml:48]: Why reintroducing jackson 1 here, as it wasn't part of bundles before PR?

Comment (andreasmattes) [reply pom.xml]: I have reverted to Jackson 1.9.12 which has been originally used by Camel, but keep the variable to have it ready if we are later asked to update for security patches.

Comment (jgreffe) [reply pom.xml]: Ok, misunderstood jackson 1 was already used in these features.

**Review by jgreffe — APPROVED** [review]
lgtm 👌🏼

**Pull request merged**